podvm/qcow2/misc-settings.sh

```diff
@@ -11,6 +11,31 @@
 # This ensures machine-id is generated during first boot and a unique
 # dhcp IP is assigned to the VM
 sudo echo -n > /etc/machine-id
-
 #Lock password for the ssh user (peerpod) to disallow logins
 sudo passwd -l peerpod
+
+# install required packages
+if [ "$CLOUD_PROVIDER" == "vsphere" ]
+then
+# Add vsphere specific commands to execute on remote
+    case $PODVM_DISTRO in
+	centos)
+	    #fallthrough
+	    ;&
+	  rhel)
+	    (! dnf list --installed | grep open-vm-tools 2>&1 >/dev/null) && \
+		(! dnf -y install open-vm-tools) && \
+		     echo "$PODVM_DISTRO: Error installing package required for cloud provider: $CLOUD_PROVIDER" 1>&2 && exit 1
+	    ;;
+	ubuntu)
```

Comment:
so Ubuntu works on vsphere now? have you tired it? IIRC on Ubuntu requires apt update before first access

Reply:
Of course, Ubuntu is the first thing we tried before RHEL. Are you seeing this with a container build ? Feel free to create an issue so we can keep track of it.